Logs.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+    <div class="container">
+        <div class="logs-container">
+            <h2>Monday-Upper</h2>
+            <div class="logs-list" id='upperData'>

```

Comment:
This is debatable but I like to use camelCase for user-defined classes. It avoids inadvertently using a reserved class name